src/components/Avatar/Avatar.react.js

```diff
@@ -5,7 +5,7 @@ import { Icon } from "../";
 import cn from "classnames";
 import AvatarList from "./AvatarList.react";
 
-type Props = {|
+export type Props = {|
```

Comment:
I can't seem to find where we are importing this

Reply:
It's not actually used in this PR, slipped in from some other stuff I was working on